jsapp/scss/stylesheets/partials/form_builder/_card_multichoice.scss

```diff
@@ -21,133 +21,146 @@
+  margin-left: 4px;
+  position: relative;
+
+  div.editable-wrapper {
+    font-weight: 600;
+    border-radius: 5px;
+    border: $thinBorderStyle;
+    padding: 5px;
+    font-size: 13px;
+    display: inline-block;
+    width: 72%;
+    vertical-align: top;
+    height: auto;
     position: relative;
-    div.editable-wrapper {
-        font-weight: 600;
-        border-radius: 5px;
-        border:$thinBorderStyle;
-        padding: 5px;
-        font-size: 13px;
-        display: inline-block;
-        width: 72%;
-        vertical-align: top;
-        height: auto;
-        position: relative;
-        min-height: 18px;
-        cursor:move;
-        span {cursor:text;}
-        .editable-input input {
-            line-height: 1em;
-            vertical-align: middle;
-            margin-top:-3px;
-        }
+    min-height: 18px;
+    cursor: move;
+
+    span {
+      cursor: text;
+    }
+
+    .editable-input input {
+      line-height: 1em;
+      vertical-align: middle;
+      margin-top: -3px;
+    }
+  }
+
+  code {
+    font-weight: 600;
+    font-size: 11px;
+    background-color: #E3E7EC;
+    padding: 2px;
+    margin: 3px;
+    width: 25%;
+    text-align: left;
+    display: inline-block;
+    margin-left: 2%;
+    overflow: hidden;
+
+    label {
+      width: 32%;
+      padding-left: 4px;
+      color: #7E858C;
+    }
+
+    span, .editable-container {
+      display: inline-block;
+      float: right;
+      width: 64%;
+      padding-right: 4px;
+      color: #7E858C;
+      text-align: right;
+      cursor: text;
     }
-    code {
-        font-family: 'Open Sans';
-        font-weight: 600;
-        font-size: 11px;
-        background-color: #E3E7EC;
-        padding:2px;
-        margin:3px;
-        width: 25%;
-        text-align: left;
-        display: inline-block;
-        margin-left: 2%;
-        overflow: hidden;
-        label {
-            width: 32%;
-            padding-left: 4px;
-            color: #7E858C;
-        }
-        span, .editable-container {
-            display: inline-block;
-            float: right;
-            width: 64%;
-            padding-right: 4px;
-            color: #7E858C;
-            text-align: right;
-            cursor: text;
-        }
-        .editable-unsaved {
-            color: $linkColor;
-        }
-        .editable-buttons {
-            display: none; // TEMPORARY
-        }
-        .editable-container .editable-input input {
-            text-align: right;
-            display: inline-block;
-            float: right;
-            color: $linkColor;
-            line-height: 1em;
-            font-size: 11px;
-            margin-top: 4px;
-        }
+
+    .editable-unsaved {
+      color: $linkColor;
     }
-    i.fa-trash-o {
-        position: absolute;
-        left:-20px;
-        top:8px;
-        color: #D08B8C;
-        font-size: 16px;
-        cursor: pointer;
+
+    .editable-buttons {
+      display: none;
+
+      // TEMPORARY
+
     }
```

Comment:
Maybe instead of:

```scss
.editable-buttons {
  display: none;

  // TEMPORARY

}
```

this would be more clear:
```scss
// TEMPORARY
.editable-buttons {display: none;}
// ENDTEMPORARY
```